Form/ChoiceList/PhpcrOdmQueryBuilderLoader.php

```diff
@@ -84,6 +93,7 @@ public function getEntitiesByIds($identifier, array $values)
         }
```

Comment:
I think there is no reason to use the QB here. Should the QB support this? I can see it would be easy to add support for the UUID, as that is a PHPCR property. but don't think there is a good way to do an analogue of "WHERE path IN('/foo', '/foobar', '/foo/bar')`